src/reducers/mainPageReducer.js

```diff
@@ -0,0 +1,7 @@
+const initialState = {
+
+};
+
+export default (state = initialState, action) => {
+
```

Comment:
If this reducer will not return anything this could potentially lead to errors. It should return state if there is no logic implemented yet.